frontend/src/components/Register.tsx

```diff
@@ -0,0 +1,88 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { useState } from 'react';
+import { Link } from "react-router-dom";
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+const Register = () => {
+    const [name, setName] = useState();
+    const [email, setEmail] = useState();
+    const [password, setPassword] = useState();
+    const navigate = useNavigate();
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        
+        axios.post( 'http://localhost:3001/register', {name, email, password})
+        .then(result => {
+            console.log(result);
+            if(result.data === "Already registered"){
```

Comment:
This is not how the backend responds to duplicate emails. The backend responds with a 409 Conflict status code. This means axios.post will throw an error. Please handle this properly.